### ISSUE TITLE
Add official MetNet-2 Version

### ISSUE THREAD

**Issue by jacobbieker** [openclimatefix/metnet#47]
https://colab.research.google.com/github/google/ai-weather-climate/blob/main/metnet2/colab.ipynb#scrollTo=Ruv8nb6E3rho

This is apparently MetNet-2 official version. Would be good to compare and incorporate that into this version. Their version is JAX-based.

**Comment by kruti107** [comment]
Hello sir,
i am kruti pandya i wanted to contribute in this issue.

**Comment by kruti107** [comment]
Please let me know about the dataset i am not getting from where i can download

**Comment by kruti107** [comment]
ok so we need to do changes in the model based on official menet 2 version got it. i would like to work in this issue.

**Comment by jacobbieker** [comment]
Yeah, happy for you to work on it! Just open a PR when you think its ready

**Comment by kruti107** [comment]
ok sure sir
